### PR TITLE
update graph style to fit new brand

### PR DESCRIPTION
**Pull request by nsolerieu**
Add new styles and dots on the lines
graph-email

### CODE REVIEW

File: server.js
Comment (visnup) [inline server.js]: if you need different colors to be on-brand, you'll need to also coordinate a server-side change to the emails since this won't change what's requested there.

File: server.js
Comment (visnup) [inline server.js]: can you remove these `console.log`s?

**Review by visnup — CHANGES_REQUESTED** [review]
need to remove the `console.log`s

File: server.js
Comment (nsolerieu) [reply server.js]: Who should I ask to do the server side change?

Comment (nsolerieu) [reply server.js]: @soul2197 Can you change the call on the server-side to use the new set of colors `1C85E8,D0D3E0` ?

Comment (ghost) [reply server.js]: @nsolerieu i will show you where you can change the colors so you can easily make changes to them: https://github.com/opendoor-labs/web/blob/master/app/commands/marketing_automation/consolidate_home_valuation_report_mailer_content.rb

Comment (nsolerieu) [reply server.js]: @soul2197 let's merge this one if it look good to you and then https://github.com/opendoor-labs/web/pull/10706

Comment (ghost) [reply server.js]: i will defer to @visnup on this one since i don't understand the d3 stuff well, once he signs off on this we can merge both

**Review by visnup — APPROVED** [review]

**Pull request merged**
